http_client.rst

```diff
@@ -854,14 +854,26 @@ To leverage all these design benefits, the cURL extension is needed.
 Enabling cURL Support
 ~~~~~~~~~~~~~~~~~~~~~
 
-This component supports both the native PHP streams and cURL to make the HTTP
-requests. Although both are interchangeable and provide the same features,
-including concurrent requests, HTTP/2 is only supported when using cURL.
+This component supports the native PHP streams, `AmpHttpClient`_ and cURL to
```

Comment:
shouldn't this say `amphp/http-client` instead as it refers to what can be used as implementation ?